src/webnn_native/ops/Unary.cpp

```diff
@@ -19,19 +19,24 @@
 
 namespace webnn_native { namespace op {
 
-    MaybeError Unary::Validate() {
-        MaybeError maybeError = OperatorBase::Validate();
+    MaybeError Unary::ValidateAndInferOutputInfo() {
+        MaybeError maybeError = OperatorBase::ValidateAndInferOutputInfo();
         if (maybeError.IsError()) {
             return maybeError;
         }
 
-        if (mOpType == UnaryOpType::kSoftmax) {
-            auto input = mInputs[0];
-            if (input->Rank() != 2) {
-                return DAWN_VALIDATION_ERROR("Input dimensions is incorrect.");
+        if (!mInputs.empty()) {
```

Comment:
return error if it's empty.

Reply:
Unary operators such as relu/hardSwich will have empty inputs. How about disabling the validation for operators in GraphBuilder?

Reply:
How to build the graph for the empty input?

Reply:
Thanks, I have removed the empty shape validation.